jupyterhub/schema.yaml

```
@@ -276,6 +276,20 @@ properties:
 
           See the [Kubernetes docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
           to learn more about labels.
+      initContainers:
+        type: list
+        description: |
+          list of initContainers to be run with hub pod. See [Kubernetes Docs](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
+          
+          ```yaml
+            initContainers:
```

```suggestion
          hub:
            initContainers:
```

This may make it more apparent where this configuration is located.